index.html

```diff
@@ -0,0 +1,152 @@
+                        <li class="list-inline "><a href="#" class="text-decoration-none nav-link">Join</a></li>
+                        <li class="list-inline "><a href="#" class="text-decoration-none nav-link">Sponser</a></li>
+                        <li class="list-inline "><a href="#" class="text-decoration-none nav-link">News</a></li>
+                        <a href="#" class="p-2 list-inline text-decoration-none border border-5 border-danger"><b class="bg-black text-muted align-content-center p-1">Buy Ticket</b></a>
+                    </ul>
+                </div>
+            </navbar>
+        </section>
+            
+            <ul class="menu">
+                <li class="nav-item"><a href="about.html" class="nav-link ">About</a></li>
+                <li class="nav-item"><a href="#" class="nav-link">Program</a></li>
+                <li class="nav-item"><a href="#" class="nav-link">Join</a></li>
+                <li class="nav-item"><a href="#" class="nav-link">Sponser</a></li>
+                <li class="nav-item"><a href="#" class="nav-link">News</a></li>
+            </ul>
+    
+
+        <section class="headline d-flex flex-column gap-1 justify-content-between  align-items-baseline ">
+            <span class="size font-color text-xl font-medium text-md-3xl font-monospace ">"Hello! Tech Lover"</span>
+            <div><h1 class=" h-grd">The TECH-No GLOBAL<br>ANNUAL EVENT 2023</h1></div>
+            <p class="p-font  border border-3 border-light p-3">In our day-to-day life technology is very useful and important. Furthermore, it <br>has made communication
+                 much easier than ever before. The introduction<br> of modified and advanced innovations of phones and its application has made<br> connecting to people much easier</p>
+                <span class="p-font"> 2023.12.30(THU) ~ 16(Sat) </span>
+                  <p class="mt-2 p-font" >  @National Musem of korean art, central Nabi and more </p>
+        </section>
+        </div>
+        <section class="program-section container-fluid bg-dark mb-5" >
+            <div class="d-flex flex-column justify-content-center align-item-center m-lg-auto p-lg-5 m-sm-1 p-sm-1">
+                <h4 class="text-center text-white p-sm-4 font-meduim pt-5">Main Program</h4>
+                <div class="container-fluid program d-flex  flex-md-row flex-lg-row justify-content-center align-items-baseline m-lg-auto p-lg-5 gap-1" >
+                   <div class="p-content bg-color d-flex flex-lg-column  flex-md-column gap-1 justify-content-around align-items-center  p-4">
+                    <img src="imgs/icons4.png" width="44px" alt="first program icon">
+                    <h6 class="font-color">Lecture</h6>
+                    <p class="text-white">The director of Ex-investor will begin by introducing the history
+                        of the event, explaining the initial idea and how we achieved
+                        success.</p>
+                   </div>
```

Comment:
### In your main program in mobile version here is some details to fix:

1. Please add more gaps between cards here to let the cards not close to each others like this ✅
2. Please add more margin/padding to the card itslef since the text is above the border of the card ✅
3. please align the items for the cards `the icon the heading and the p tag` to be centered to match the given design ✅

- Your design
![image](https://user-images.githubusercontent.com/58553711/227082500-dcc7f8a0-222c-4917-96b8-489c96f1634a.png)


- Given
![image](https://user-images.githubusercontent.com/58553711/227082632-7744e596-716a-4740-b8fd-4863f7c15b09.png)